pages/src/_data/buildinginfo.yml

```diff
@@ -31,6 +33,7 @@ bakery:
           description: The hut will look in the warehouse first to see what resource you have more of before deciding what recipe it will use.
 barracks:
   name: Barracks
```

Comment:
Fun fact: the singular of "barracks" is "barrack."
But I almost never hear anyone refer to a single barrack.